### PR TITLE
Add punyo model

### PR DESCRIPTION
**Pull request by aykut-tri**
- Adds Punyo model w/ floaties
- Adds the box with lid as a manipuland

### CODE REVIEW

File: models/punyo_arm.sdf [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+        <material>
+          <diffuse>0.8 0.8 0.8 1.0</diffuse>
+        </material>
+      </visual>
+      <collision name='floatie_1.0_on_links_5_and_6_collision'>
+        <pose>0 0 0 1.507 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.12</length>
+            <radius>0.06</radius>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:500]: from my measurements, this should be 0.065. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+        <material>
+          <diffuse>0.8 0.8 0.8 0.7</diffuse>
+        </material>
+      </visual>
+      <collision name='two_floatie_2.0s_on_link_4_collision'>
+        <pose>0 0.125 0 1.507 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.1</length>
+            <radius>0.09</radius>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:337]: from my measurements, this should be 0.85. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 3]
```diff
@@ -0,0 +1,643 @@
...
+        </geometry>
+        <material>
+          <diffuse>0.8 0.8 0.8 0.7</diffuse>
+        </material>
+      </visual>
+      <collision name='two_floatie_2.0s_on_link_4_collision'>
+        <pose>0 0.125 0 1.507 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.1</length>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:336]: from my measurements, this should be 0.3. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 4]
```diff
@@ -0,0 +1,643 @@
...
+        </geometry>
+        <material>
+          <diffuse>0.8 0.8 0.8 0.7</diffuse>
+        </material>
+      </visual>
+      <collision name='two_floatie_2.0s_on_link_3_collision'>
+        <pose>0 0 -0.085 0 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.1</length>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:245]: from my measurements, this should be 0.12. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 5]
```diff
@@ -0,0 +1,643 @@
...
+            <radius>0.09</radius>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:246]: from my measurements, this should be 0.075. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 6]
```diff
@@ -0,0 +1,643 @@
...
+        <material>
+          <diffuse>0.627451 0.627451 0.627451 0.8</diffuse>
+        </material>
+      </visual>
+
+      <visual name='two_floatie_2.0s_on_link_2_visual'>
+        <pose>0 -0.125 0 1.507 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.1</length>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:142]: from my measurements, this should be 0.12. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 7]
```diff
@@ -0,0 +1,643 @@
...
+            <radius>0.09</radius>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:143]: from my measurements, this should be 0.075. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 8]
```diff
@@ -0,0 +1,643 @@
...
+            <uri>package://drake/manipulation/models/jaco_description/meshes/ring_big.obj</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>0.627451 0.627451 0.627451 0.8</diffuse>
+        </material>
+      </visual>
+
+      <visual name='two_floatie_2.0s_on_link_3_visual'>
+        <pose>0 0 -0.085 0 0 0</pose>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:230]: <pose>0 0 -0.065 0 0 0</pose>. Fix collision below as well.

File: models/punyo_arm.sdf [hunk 9]
```diff
@@ -0,0 +1,643 @@
...
+            <uri>package://drake/manipulation/models/jaco_description/meshes/ring_big.obj</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>0.627451 0.627451 0.627451 0.8</diffuse>
+        </material>
+      </visual>
+
+      <visual name='two_floatie_2.0s_on_link_2_visual'>
+        <pose>0 -0.125 0 1.507 0 0</pose>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:139]: <pose>0 -0.14 0 1.507 0 0</pose>. Fix collision below as well.

File: models/punyo_arm.sdf [hunk 10]
```diff
@@ -0,0 +1,643 @@
...
+        </geometry>
+        <material>
+          <diffuse>0.8 0.8 0.8 1.0</diffuse>
+        </material>
+      </visual>
+      <collision name='floatie_1.0_on_links_5_and_6_collision'>
+        <pose>0 0 0 1.507 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.12</length>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:499]: from my measurements, this should be 0.11. Fix visual above as well.

File: models/punyo_arm.sdf [hunk 11]
```diff
@@ -0,0 +1,643 @@
...
+            <radius>0.06</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <diffuse>0.8 0.8 0.8 1.0</diffuse>
+        </material>
+      </visual>
+      <collision name='floatie_1.0_on_links_5_and_6_collision'>
+        <pose>0 0 0 1.507 0 0</pose>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:496]: <pose>0 0.09 0 1.507 0 0</pose>. Five visual as well.

File: models/punyo_arm.sdf [hunk 12]
```diff
@@ -0,0 +1,643 @@
...
+            <length>0.1</length>
+            <radius>0.09</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <diffuse>0.8 0.8 0.8 0.7</diffuse>
+        </material>
+      </visual>
+      <collision name='two_floatie_2.0s_on_link_4_collision'>
+        <pose>0 0.125 0 1.507 0 0</pose>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:333]: <pose>0 0.14 -0.01 1.6 0 0</pose>

File: models/punyo_arm.sdf [hunk 13]
```diff
@@ -0,0 +1,643 @@
...
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>package://drake/manipulation/models/jaco_description/meshes/ring_small.obj</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>0.627451 0.627451 0.627451 0.8</diffuse>
+        </material>
+      </visual>
+    </link>
```
Comment (JoseBarreiros-TRI) [inline models/punyo_arm.sdf:410]: add collisions for link 5

```
      <visual name='two_floatie_2.0s_on_link_5_visual'>
        <pose>0. 0.005 -0.03 0.01 0 0</pose>
        <geometry>
          <cylinder>
            <length>0.04</length>
            <radius>0.075</radius>
          </cylinder>
        </geometry>
        <material>
          <diffuse>0.8 0.8 0.8 0.7</diffuse>
        </material>
      </visual>
      <collision name='two_floatie_2.0s_on_link_5_collision'>
        <pose>0. 0.005 -0.03 0.01 0 0</pose>
        <geometry>
          <cylinder>
            <length>0.04</length>
            <radius>0.075</radius>
          </cylinder>
        </geometry>
        <surface>
          <friction>
          <ode>
            <mu>1.0</mu>
            <mu2>1.0</mu2>
          </ode>
          </friction>
        </surface>
      </collision>
```

**Review by JoseBarreiros-TRI — CHANGES_REQUESTED** [review]
just a few nits with the collision geometries. The ones suggested are from measurements on the real robot. The collision geometries are smaller than the floaties considering compression of ~1.5 cm.

File: models/punyo_arm.sdf (hunk 1 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf:500]: done, thanks

File: models/punyo_arm.sdf (hunk 2 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 3 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: set to 0.13 per f2f convo, thanks

File: models/punyo_arm.sdf (hunk 4 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 5 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 6 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 7 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 8 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 9 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 10 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 11 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: done, thanks

File: models/punyo_arm.sdf (hunk 12 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf]: set to `0 0.13 -0.01 1.6 0 0` instead since the length is 0.13 m as well, thanks

File: models/punyo_arm.sdf (hunk 13 above)
Comment (aykut-tri) [reply models/punyo_arm.sdf:410]: not done per f2f

**Comment by aykut-tri** [comment]
Thanks a lot for all the measurements! Ready for the second pass...

**Comment by JoseBarreiros-TRI** [comment]
LGTM, it is ready to merge.

**Pull request merged**